### PR TITLE
html-forms | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
image

image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div>Favorite Language (choose one)</div>
+      <input type="radio" name="Language" value="HTML" /> HTML
+      <input type="radio" name="Language" value="CSS" /> CSS
+      <input type="radio" name="Language" value="JavaScript" /> JavaScript
+    </label>
+    <label>
+      <div>Would you like to sign up for more information?</div>
+      <input type="checkbox" name="Sign me up" value="Sign me up" /> Sign me up!
+    </label>
+    <label>
```
Comment (codingcodymiller) [inline html-forms/index.html:34]: You shouldn't wrap everything here in a `label` element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <div>Favorite Language (choose one)</div>
+      <input type="radio" name="Language" value="HTML" /> HTML
+      <input type="radio" name="Language" value="CSS" /> CSS
+      <input type="radio" name="Language" value="JavaScript" /> JavaScript
+    </label>
+    <label>
+      <div>Would you like to sign up for more information?</div>
+      <input type="checkbox" name="Sign me up" value="Sign me up" /> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:32]: The text labelling the checkbox should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    </label>
+    <label>
+      <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <textarea name="comments" cols="28" rows="9"></textarea>
+    </label>
+    <label>
```
Comment (codingcodymiller) [inline html-forms/index.html:38]: This `label` element can be removed

**Comment by rocky-truong** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**